src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
you can use `for...of` to keep it simple
```suggestion
  for (const action in actions) {
```